prompt_toolkit/output/vt100.py

```diff
@@ -432,6 +433,7 @@ def __init__(
         self.default_color_depth = default_color_depth
         self._get_size = get_size
         self.term = term
+        self.beep = bell
```

Comment:
Can you call this ``enable_bell``? Both for the attribute and the parameter?
(I see it's called ``beep`` here to avoid collision with the method.)